<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="./style.css" />
    <title>Duschenne Muscular Dystrophy Drug Target Research</title>
  </head>
  <body>
    <h1 style="color: #710202">
      Identification of Potential Drug Targets for Duchenne Muscular Dystrophy
      (DMD)
    </h1>
    <section id="research-question">
      <h2>Research Question</h2>
      <p>
        How do the interactions between small molecules and proteins coded by
        dysregulated genes in Duchenne Muscular Dystrophy (DMD) influence the
        biological processes and pathways implicated in muscle degeneration and
        weakness, and how can these interactions be leveraged to develop novel
        therapeutic interventions for DMD?
      </p>
    </section>

    <section id="DMD-explanation">
      <h2>What is Duschenne Muscular Dystrophy:</h2>
      <p>
        Duchenne Muscular Dystrophy (DMD) is a hereditary condition marked by
        continual muscle weakening and atrophy. It is classified among various
        muscular dystrophies and stems from a deficiency of dystrophin, a vital
        protein that maintains the structural integrity of muscle cells.
        Typically, DMD manifests in young children, often becoming noticeable by
        the time they reach 2 to 3 years old (Muscular Dystrophy Association,
        2023).
      </p>
      <img
        src="https://medlineplus.gov/images/PX000110_PRESENTATION.jpeg"
        alt="DMD Comparison"
      />
      <p style="text-align: center">Image: (MedLinePlus, 2023)</p>
    </section>

    <section id="wikipathways-image">
      <h2>WikiPathways Diagram(WP5356)</h2>
      <p>
        Note: This following image are taken from WikiPathways,
        <i>WPID = 5356</i>
      </p>
      <img
        src="https://www.wikipathways.org/wpi/wpi.php?action=downloadFile&type=png&pwTitle=Pathway:WP5356"
        alt="DMD Diagram"
      />
      <p>
        1. Mitochondrial Dysfunction: Dystrophin deficiency can disrupt the
        normal function of mitochondria, leading to reduced energy production
        and increased oxidative stress. This can cause further damage to muscle
        cells and contribute to the pathology of DMD.
        <br /><br />
        2. Impaired Muscle Contraction: Without dystrophin, the muscle fibers
        are more susceptible to damage during contraction. Over time, this
        damage accumulates, leading to the weakening and loss of muscle tissue.
        <br /><br />
        3. Inflammatory Pathway: Damaged muscle fibers release signals that
        activate inflammatory cells, leading to chronic inflammation that
        further damages muscle tissue and impairs regeneration.
        <br /><br />
        4. Fibrosis Pathway: In response to chronic muscle damage and
        inflammation, fibrotic tissue replaces the normal muscle cells. This
        fibrosis contributes to the stiffness and eventual loss of muscle
        function characteristic of DMD.
      </p>
    </section>

    <section id="explanation">
      <h2>Explanation:</h2>
      <p>
        Identifying potential drug targets for Duchenne Muscular Dystrophy (DMD)
        is an important goal in medicine and drug development. DMD is a genetic
        disease characterized by muscle degeneration and weakness. Understanding
        the biological processes and pathways associated with dysregulated genes
        in DMD is critical for targeting the underlying molecular mechanisms. In
        addition, knowledge of which small molecules interact with the proteins
        encoded by these genes may pave the way for drug repurposing and
        therapeutic interventions to reduce the symptoms of the disease.
        <br /><br />
        Upregulated genes in DMD vs control (human, PMID:12415109) =
        <i
          >C14ORF132, LAPTM5, DCLK1, LGALS3, OLFML2A, TMEM158, BGN, LIPA, SGSH,
          HSPG2, WWTR1, CNOT1, SERPINE2, GPC3, SYNE2, ARPC1B, MRC2, MEG3,
          COL1A2, TUBA1B, FYN, MYH3, MAGED1, VCAN, IFFO1, CD99, MYL6B, NDN,
          CLIC4, DBN1, NNMT, CASQ2, ACTN1, AHSA2, HLA-DPB1, LUM, TSPO, ANXA1,
          PCOLCE, PXDN, ACTG1, LRRC17, APOC1, TRO, RUNX1, MYOF, CFHR1,
          EZRMYH8,</i
        >
      </p>
    </section>

    <div>
      <section id="genetable">
        <h1><u>Working Gene Tables Information</u></h1>
        <p>This are the genes that...</p>
        <table border="1">
          <tr bgcolor="grey">
            <th>Number</th>
            <th>Name</th>
            <th>ID</th>
            <th>Target Type</th>
            <th>Preferred Name</th>
            <th>Protein Target Classification</th>
            <th>All Results</th>
          </tr>

          <tr bgcolor="lightgrey">
            <td>1</td>
            <td>MYOF</td>
            <td>CHEMBL4523476</td>
            <td>SINGLE PROTEIN</td>
            <td>Myoferlin</td>
            <td>Unclassified Protein</td>
            <td>2</td>
          </tr>

          <tr bgcolor="lightgrey">
            <td>2</td>
            <td>RUNX1</td>
            <td>CHEMBL2093862</td>
            <td>PROTEIN-PROTEIN INTERACTION</td>
            <td>
              Runt-related transcription factor 1/Core-binding factor subunit
              beta
            </td>
            <td>Transcription Factor</td>
            <td>10000</td>
          </tr>
          <tr bgcolor="lightgrey">
            <td>3</td>
            <td>TSPO</td>
            <td>CHEMBL5742</td>
            <td>SINGLE PROTEIN</td>
            <td>Translocator Protein</td>
            <td>Membrane Receptor</td>
            <td>872</td>
          </tr>

          <tr bgcolor="lightgrey">
            <td>4</td>
            <td>NNMT</td>
            <td>CHEMBL2346486</td>
            <td>SINGLE PROTEIN</td>
            <td>Nicotinamide N-methyltransferase</td>
            <td>Transferase, Enzyme</td>
            <td>1023</td>
          </tr>

          <tr bgcolor="lightgrey">
            <td>5</td>
            <td>MAGED1</td>
            <td>CHEMBL4742325</td>
            <td>PROTEIN-PROTEIN INTERACTION</td>
            <td>Cereblon/Melanoma-associated antigen D1</td>
            <td>Unclassified Protein</td>
            <td>2</td>
          </tr>

          <tr bgcolor="lightgrey">
            <td>6</td>
            <td>FYN</td>
            <td>CHEMBL1841</td>
            <td>SINGLE PROTEIN</td>
            <td>Tyrosine-protein kinase FYN</td>
            <td>
              Enzyme, Kinase, Protein Kinase, Tyrosine protein kinase Src family
              , TK protein kinase group
            </td>
            <td>10000</td>
          </tr>

          <tr bgcolor="lightgrey">
            <td>7</td>
            <td>TUBA1B</td>
            <td>CHEMBL3797010</td>
            <td>SINGLE PROTEIN</td>
            <td>Tubulin alpha-1B chain</td>
            <td>Structural protein</td>
            <td>4</td>
          </tr>

          <tr bgcolor="lightgrey">
            <td>8</td>
            <td>COL1A2</td>
            <td>CHEMBL1841</td>
            <td>SINGLE PROTEIN</td>
            <td>Collagen alpha 2 chain</td>
            <td>Structural Protein</td>
            <td>8</td>
          </tr>

          <tr bgcolor="lightgrey">
            <td>9</td>
            <td>LIPA</td>
            <td>CHEMBL4184</td>
            <td>SINGLE PROTEIN</td>
            <td>Lysosomal acid lipase/cholesteryl ester hydrolase</td>
            <td>Hydrolase, Enzyme</td>
            <td>60</td>
          </tr>

          <tr bgcolor="lightgrey">
            <td>10</td>
            <td>CNOT1</td>
            <td>CHEMBL4105919</td>
            <td>SINGLE PROTEIN</td>
            <td>CCR4-NOT transcription complex subunit 1</td>
            <td>Unclassified protein</td>
            <td>0</td>
          </tr>

          <tr bgcolor="lightgrey">
            <td>11</td>
            <td>GPC3</td>
            <td>CHEMBL4630889</td>
            <td>SINGLE PROTEIN</td>
            <td>Glypican-3</td>
            <td>Unclassified protein</td>
            <td>0</td>
          </tr>

          <tr bgcolor="lightgrey">
            <td>12</td>
            <td>ARPC1B</td>
            <td>CHEMBL4295656</td>
            <td>SINGLE PROTEIN</td>
            <td>Actin-related protein 2/3 complex subunit 1B</td>
            <td>Unclassified protein</td>
            <td>0</td>
          </tr>

          <tr bgcolor="lightgrey">
            <td>13</td>
            <td>DCLK1</td>
            <td>CHEMBL5683</td>
            <td>SINGLE PROTEIN</td>
            <td>Serine/threonine-protein kinase DCLK1</td>
            <td>
              Protein Kinase, CAMK protein kinase group, CAMK protein kinase
              DCAMK1 family, Kinase, Enzyme
            </td>
            <td>6012</td>
          </tr>

          <tr bgcolor="lightgrey">
            <td>14</td>
            <td>LGALS3</td>
            <td>CHEMBL4531</td>
            <td>SINGLE PROTEIN</td>
            <td>Galectin-3</td>
            <td>Other cytosolic protein</td>
            <td>958</td>
          </tr>
        </table>
        <ul id="geneDesc">
          <li>
            <b>#1 MYOF (CHEMBL4523476)</b>: Myoferlin is a protein that plays a
            significant role in muscle repair and regeneration. It is essential
            for the maintenance of muscle integrity and has been implicated in
            muscular dystrophies due to its involvement in muscle cell membrane
            repair.
          </li>
          <li>
            <b>#2 RUNX1 (CHEMBL2093862)</b>: This gene encodes a protein that is
            a member of the runt domain-containing family of transcription
            factors. RUNX1 is crucial for hematopoietic stem cell formation and
            is involved in the transcriptional regulation of genes important for
            cell lineage differentiation and proliferation. Its role in DMD may
            be linked to inflammatory processes or muscle repair mechanisms that
            are active in the disease state.
          </li>
          <li>
            <b>#3 TSPO (CHEMBL5742)</b>: The TSPO gene codes for the
            translocator protein, which is located in the outer mitochondrial
            membrane. This protein is involved in the transport of cholesterol
            into mitochondria and is thought to play a role in steroidogenesis,
            cell proliferation, and apoptosis. It has been of interest in
            various research areas due to its association with tissue injury and
            inflammation, which are key factors in the pathology of DMD.
          </li>
          <li>
            <b>#4 NNMT (CHEMBL2346486)</b>: Nicotinamide N-methyltransferase is
            an enzyme involved in the biotransformation of many drugs and
            xenobiotic compounds. It catalyzes the N-methylation of nicotinamide
            and other pyridines. It is also implicated in several metabolic
            disorders and cancers due to its role in the methylation of biogenic
            amines.
          </li>
          <li>
            <b>#5 MAGED1 (CHEMBL4742325)</b>: Melanoma-associated antigen D1 is
            a protein that may be involved in cell cycle progression, apoptosis,
            and transcriptional regulation. It has been studied for its
            potential role in cancer as it is differentially expressed in a
            variety of malignancies.
          </li>
          <li>
            <b>#6 FYN (CHEMBL1841)</b>: FYN is a gene that encodes a member of
            the Src family of tyrosine kinases. This protein is involved in the
            signaling pathways that regulate cell growth, differentiation, and
            migration. Its kinase activity is crucial for various cellular
            processes and is implicated in the signaling pathways that might
            affect muscle cell integrity and function in DMD.
          </li>
          <li>
            <b>#7 TUBA1B (CHEMBL3797010)</b>: Tubulin alpha-1B chain is a
            fundamental component of microtubules, which are cylindrical fibers
            that are part of the cytoskeleton in all eukaryotic cells. Tubulins
            play critical roles in cell structure, intracellular transport, and
            mitosis.
          </li>
          <li>
            <b>#8 COL1A2 (CHEMBL1841)</b>: Collagen alpha-2(I) chain is part of
            type I collagen, the fibrillar collagen found in most connective
            tissues, including cartilage. Mutations in this gene are associated
            with osteogenesis imperfecta, Ehlers-Danlos syndrome, and other
            disorders of connective tissue.
          </li>
          <li>
            <b>#9 LIPA (CHEMBL4184)</b>: Lysosomal acid lipase/cholesteryl ester
            hydrolase is an enzyme that hydrolyzes cholesteryl esters and
            triglycerides in lysosomes. Deficiency in this enzyme leads to the
            accumulation of cholesteryl esters and triglycerides, causing
            lysosomal storage diseases such as Wolman disease and cholesteryl
            ester storage disease.
          </li>
          <li>
            <b>#10 CNOT1 (CHEMBL4105919)</b>: CNOT1 is a part of the CCR4-NOT
            complex, which is a global regulator of gene expression and is
            implicated in various processes including mRNA turnover, regulation
            of gene expression, and cell proliferation. The CCR4-NOT complex is
            crucial for maintaining cellular homeostasis and its dysfunction is
            linked to various diseases including cancer.
          </li>
          <li>
            <b>#11 GPC3 (CHEMBL4630889)</b>: Glypican 3 is a member of the
            glypican family, which are heparan sulfate proteoglycans attached to
            the cell surface. GPC3 is involved in the regulation of cell growth
            and division and is particularly important during development.
            Mutations in this gene are associated with Simpson-Golabi-Behmel
            syndrome, a condition characterized by pre- and postnatal overgrowth
            with developmental delay and other anomalies.
          </li>
          <li>
            <b>#12 ARPC1B (CHEMBL4295656)</b>: Actin-related protein 2/3 complex
            subunit 1B is a component of the Arp2/3 complex which plays a
            pivotal role in the regulation of the actin cytoskeleton. The Arp2/3
            complex is involved in the control of cell shape, motility, and
            adhesion. Mutations in ARPC1B are linked to immune system disorders
            and platelet abnormalities.
          </li>
          <li>
            <b>#13 DCLK1 (CHEMBL5683)</b>: Doublecortin-like kinase 1 is a
            microtubule-associated protein expressed in neuronal precursor cells
            and is thought to be involved in neurogenesis. It is characterized
            by its doublecortin domain and is associated with microtubule
            polymerization, playing a role in neuronal migration and
            differentiation. It has also been studied for its potential
            involvement in various cancers.
          </li>
          <li>
            <b>#14 LGALS3 (CHEMBL4531)</b>: Galectin-3 is a protein that is part
            of the lectin family, which can bind beta-galactoside sugars. It is
            involved in various cellular functions, including cell adhesion,
            cell cycle regulation, immune response, apoptosis, and metastasis.
            Elevated levels of galectin-3 have been observed in cancer and it is
            being explored as a potential biomarker and therapeutic target for
            various types of cancer.
          </li>
        </ul>
      </section>
    </div>

    <div class="code-container">
      <section>
        <h2>SparQL Query Code Samples</h2>
        <h3>Return Property & Value with Associated Resource/Target:</h3>
        <pre><code class="language-sql">
        PREFIX chembl: &lt;http://rdf.ebi.ac.uk/terms/chembl#&gt;

        SELECT ?Property ?Value WHERE {
          target:CHEMBL2685 ?Property ?Value .
        }
        LIMIT 1000
      </code></pre>
        <button class="copy-button" onclick="copyToClipboard(this)">
          Copy Code
        </button>
      </section>
    </div>

    <div class="code-container">
      <section>
        <h3>Protein Classiication Query:</h3>
        <pre><code class="language-sql">
        PREFIX chembl: &lt;http://rdf.ebi.ac.uk/terms/chembl#&gt;
        PREFIX target: &lt;http://rdf.ebi.ac.uk/resource/chembl/target/&gt;
        
        SELECT ?target ?targetLabel ?proteinClassLabel ?targetType WHERE {
          VALUES ?target {target:CHEMBL4523476}
          ?target rdfs:label ?targetLabel.
          ?target chembl:hasProteinClassification ?proteinClass .
          ?proteinClass rdfs:label ?proteinClassLabel.
          
          OPTIONAL {
            ?target chembl:targetType ?targetType .
        }
        } LIMIT 10000
      </code></pre>
        <button class="copy-button" onclick="copyToClipboard(this)">
          Copy Code
        </button>
      </section>
    </div>

    <div class="code-container">
      <section>
        <h3>
          All-inclusive Result Query(includes every binding affinity types, not
          limited):
        </h3>
        <pre><code class ="language-sql">
          PREFIX chembl: &lt;http://rdf.ebi.ac.uk/terms/chembl#&gt;
          PREFIX target: &lt;http://rdf.ebi.ac.uk/resource/chembl/target/&gt;

          SELECT DISTINCT ?target ?targetLabel ?molLabel ?assayLabel ?type ?value ?proteinClassLabel WHERE {
           VALUES ?target {target:CHEMBL4523476}
            ?assay  chembl:hasTarget ?target.
 
            ?activity chembl:hasAssay  ?assay.
            ?activity chembl:hasMolecule ?molecule .

            ?target rdfs:label ?targetLabel.
            ?molecule rdfs:label ?molLabel.
            ?assay  rdfs:label ?assayLabel.

            ?activity chembl:type ?type.
            ?activity chembl:standardValue ?value.

          OPTIONAL {
            ?target chembl:hasProteinClassification ?proteinClass .
            ?proteinClass rdfs:label ?proteinClassLabel.
           }
          }
      </code></pre>
        <button class="copy-button" onclick="copyToClipboard(this)">
          Copy Code
        </button>
      </section>
    </div>

    <div class="code-container">
      <section>
        <h3>
          Binding Affinity Query(exclusive binding affinity types) Example:
        </h3>
        <pre><code class="language-sql">
        PREFIX chembl: &lt;http://rdf.ebi.ac.uk/terms/chembl#&gt;
        PREFIX target: &lt;http://rdf.ebi.ac.uk/resource/chembl/target/&gt;
        
        SELECT DISTINCT ?target ?targetLabel ?molLabel ?assayLabel ?type ?value ?proteinClassLabel WHERE {
          VALUES ?target {target:CHEMBL4523476}
          ?assay  chembl:hasTarget ?target.
       
          ?activity chembl:hasAssay  ?assay.
          ?activity chembl:hasMolecule ?molecule.
        
          ?target rdfs:label ?targetLabel.
          ?molecule rdfs:label ?molLabel.
          ?assay  rdfs:label ?assayLabel.
        
          ?activity chembl:type ?type.
          ?activity chembl:standardValue ?value.
        
          FILTER(?type IN ("Ki", "Ka", "Kd", "Kdiss" "IC50", "EC50"))
        
          OPTIONAL {
            ?target chembl:hasProteinClassification ?proteinClass .
            ?proteinClass rdfs:label ?proteinClassLabel.
          }
        } 
        ORDER BY ASC(?value) 
        LIMIT 10000
      </code></pre>
        <button class="copy-button" onclick="copyToClipboard(this)">
          Copy Code
        </button>
        <br /><br />
        <p>
          In life sciences such as pharmacology and biochemistry, Ki(inhibition
          constant) and Kd(dissociation constant) are two important parameters
          used to describe the binding affinity between a ligand (which can be a
          drug, hormone, neurotransmitter, or any molecule that binds to a
          receptor) and a protein (such as a receptor or enzyme). Here's why
          they are important:
        </p>
        <ul>
          <li>
            <b>Ki (Inhibition Constant)</b>: Ki is the concentration of an
            inhibitor (ligand) that is required to reduce the binding of a
            particular ligand to its receptor by half in the presence of a fixed
            concentration of the ligand. The Ki value is used to assess the
            potency of an inhibitor. A lower Ki means that the inhibitor is more
            effective at binding to the target protein, which in turn means that
            a lower concentration of the inhibitor is needed to achieve the
            desired inhibitory effect. This is particularly relevant in drug
            development, where high potency is often desirable.
          </li>
          <li>
            <b>Kd/Kdiss (Dissociation Constant) </b>: Kd&Kdiss are used
            interchangably, it is the concentration of ligand at which half of
            the receptor sites are occupied, reflecting the equilibrium between
            the ligand-bound and unbound receptor states. The Kd value indicates
            the affinity of the ligand for the receptor. A lower Kd value
            signifies tighter binding between the ligand and its receptor. In
            the context of drug design, a lower Kd is typically advantageous
            because it suggests that the drug will bind effectively to its
            target at lower concentrations, potentially reducing the dose needed
            and possibly resulting in fewer side effects.
          </li>
          <li>
            <b>Ka (Association Constant)</b>: This is used to describe the
            affinity of the ligand for the target. A higher Ka implies a
            stronger affinity, meaning the ligand binds more tightly to its
            target. It's the inverse of Kd (dissociation constant).
          </li>
          <li>
            <b>IC50 (Half-maximal Inhibitory Concentration)</b>: While IC50 is
            more a measure of functional inhibition than direct binding
            affinity, it can still provide useful information, especially if the
            mechanism of inhibition is known. For competitive inhibitors, IC50
            can be related to Ki under certain conditions, but this relationship
            depends on the concentration of the substrate and the
            characteristics of the enzyme.
          </li>
          <li>
            <b>EC50 (Half-maximal Effective Concentration)</b>: Similar to IC50,
            EC50 is a measure of a drug's functional effect (e.g., activation of
            a receptor) rather than direct binding affinity. However, in the
            context of agonists, it can give an indication of how effectively a
            molecule activates its target.
          </li>
        </ul>
        <div style="text-align: center">
          <i
            >A small Ki value indicates that the inhibitor binds more
            effectively to the target protein, meaning it can exert its
            inhibitory effect at a lower concentration. A low Kd/Kdiss means
            that the ligand binds tightly to the protein and remains bound with
            higher affinity. A higher Ka value signifies a greater affinity of
            the ligand for the receptor, indicating that the ligand binds more
            readily and tightly to the receptor. Overall, the most useful
            measurements to know the binding affinity are: Ki, Kd, & Ka!
          </i>
        </div>
      </section>
    </div>

    <div>
      <section id="graphical-display">
        <h2>Graphical Display</h2>
        <p>
          Our goal is to build an informative graph representing three types of
          nodes: Proteins/Genes, Small Molecules, and Protein Classification,
          along with their connections.
        </p>
      </section>
    </div>

    <div id="network-visualization"></div>
    <section>
      <script src="https://d3js.org/d3.v6.min.js"></script>
      <script src="./d3.js"></script>
      <script src="./script.js"></script>
    </section>

    <section>
      <h2>Developing novel therapeutic interventions for DMD</h2>
      <p>
        - Modulating Inflammatory Responses: Drugs that modulate the immune
        response may reduce inflammation and slow muscle damage.
        <br /><br />
        - Enhancing Muscle Repair: Therapies that promote muscle cell repair and
        regeneration could help maintain muscle function.
        <br /><br />
        - Preventing Fibrosis: Agents that prevent or reverse fibrosis could
        help preserve muscle elasticity and function.
        <br /><br />
        - Restoring Dystrophin Function: Gene therapy and exon-skipping drugs
        aim to restore the production of functional dystrophin protein in muscle
        cells.
      </p>
    </section>

    <section>
      <h2>Answers to Research Question:</h2>
      <p>
        Duchenne Muscular Dystrophy (DMD) is a genetic disorder characterized by
        progressive muscle degeneration and weakness. It is caused by mutations
        in the DMD gene, which encodes dystrophin, a protein critical for
        maintaining muscle cell integrity. The interactions between small
        molecules and proteins coded by dysregulated genes in DMD play a
        significant role in the disease's progression and offer potential
        avenues for therapeutic intervention.
      </p>
      <h3>Understanding the Molecular Interactions in DMD:</h3>
      <p>
        1. Role of Dystrophin: Dystrophin is part of a complex of proteins that
        connects the cytoskeleton of a muscle fiber to the surrounding
        extracellular matrix through the cell membrane. In DMD, mutations in the
        dystrophin gene lead to an absence or severe reduction of the dystrophin
        protein, compromising muscle fiber stability and leading to muscle
        damage and inflammation.
        <br /><br />
        2. Compensatory Mechanisms: The absence of dystrophin triggers a cascade
        of molecular events. The body attempts to compensate for the loss of
        dystrophin, which can involve the upregulation of other proteins,
        changes in signaling pathways, and activation of inflammatory responses.
        These compensatory mechanisms, while initially protective, can become
        maladaptive over time, contributing to muscle degeneration.
        <br /><br />
        3. Pathways Implicated in Muscle Degeneration: Key pathways involved in
        DMD pathology include inflammation, fibrosis, and oxidative stress.
        Additionally, dysregulation of calcium homeostasis and mitochondrial
        dysfunction are critical factors in muscle degradation.
      </p>

      <h3>Leveraging Molecular Interactions for Therapeutic Interventions:</h3>
      <p>
        Modulating Compensatory Mechanisms: Small molecules can be used to
        modulate the compensatory mechanisms. This includes anti-inflammatory
        drugs to reduce muscle inflammation, antioxidants to address oxidative
        stress, and agents that modulate calcium influx and mitochondrial
        function.
      </p>
      <h3>Interaction between Small Molecules & Target Genes:</h3>
      <p>
        The interactions between small molecules and proteins coded by
        dysregulated genes in Duchenne Muscular Dystrophy (DMD) can have
        profound effects on the biological processes and pathways that
        contribute to muscle degeneration and weakness.
        <br /><br />
        <b>1. MYOF (Myoferlin):</b> Small molecule interactions that promote or
        stabilize myoferlin function could enhance muscle cell membrane repair
        mechanisms, potentially slowing down the progression of muscle damage in
        DMD.<br /><br />

        <b>2. RUNX1:</b> Modulators of RUNX1 could influence transcriptional
        regulation, possibly affecting stem cell differentiation and potentially
        aiding in muscle repair and regeneration.<br /><br />

        <b>3. TSPO:</b> Small molecules that affect TSPO function could modulate
        cholesterol transport and mitochondrial function, impacting
        steroidogenesis, cell proliferation, and apoptosis, all relevant to the
        maintenance of muscle cell health.<br /><br />

        <b>4. NNMT:</b> By influencing NNMT activity, small molecules could
        affect the metabolism of drugs and other compounds, potentially
        ameliorating metabolic disorders that could exacerbate muscle
        degeneration.<br /><br />

        <b>5. MAGED1:</b> Targeting MAGED1 could affect cell cycle progression
        and transcriptional regulation, which could have implications for muscle
        cell regeneration and differentiation.<br /><br />

        <b>6. FYN:</b> This kinase is involved in signaling pathways that
        regulate cell growth, differentiation, and migration. Small molecule
        kinase inhibitors or activators could modulate these signaling pathways,
        potentially preserving muscle cell integrity and function.<br /><br />

        <b>7. TUBA1B:</b> Stabilizing microtubule function through TUBA1B could
        help maintain cytoskeletal integrity, important for muscle cell
        structure and intracellular transport.<br /><br />

        <b>8. COL1A2:</b> Modulating collagen expression or stability could
        influence connective tissue health, which is crucial for muscle function
        and integrity.<br /><br />

        <b>9. LIPA:</b> Small molecules that can enhance or mimic LIPA activity
        may prevent the pathological accumulation of cholesteryl esters and
        triglycerides, thereby addressing aspects of lysosomal storage diseases
        that can influence muscle pathology.<br /><br />

        <b>10. CNOT1:</b> This protein is involved in gene expression
        regulation, and modulating its activity could influence cellular
        homeostasis and response to muscle damage.<br /><br />

        <b>11. GPC3:</b> Small molecules that interact with GPC3 might modify
        cell growth and division processes, potentially ameliorating conditions
        that lead to muscle weakness.<br /><br />

        <b>12. ARPC1B:</b> Targeting the actin cytoskeleton through ARPC1B could
        influence cell shape, motility, and adhesion, all of which are essential
        in muscle cells and their ability to regenerate and repair.<br /><br />

        <b>13. DCLK1:</b> Modulating the activity of DCLK1 could influence
        neuronal precursor cells and neurogenesis, which, while more indirectly,
        could impact the overall health and repair mechanisms of muscle
        tissue.<br /><br />

        <b>14. LGALS3:</b> Galectin-3 is involved in various cellular functions,
        and modulating its activity could impact cell adhesion, apoptosis, and
        metastasis, which are relevant in the context of inflammatory responses
        in DMD muscle tissue.
      </p>
    </section>
  </body>
</html>
